distributions/standalone/src/main/resources/conf/scheduler-pools.conf

```diff
@@ -11,17 +13,26 @@ org.mule.runtime.scheduler.gracefulShutdownTimeout=15000
 # The number of threads to keep in the cpu_lite pool, even if they are idle.
 # Supports Expressions
 org.mule.runtime.scheduler.cpuLight.threadPool.size=2*cores
+# The size of the queue to use for holding cpu_lite tasks before they are executed.
+# Supports Expressions
+org.mule.runtime.scheduler.cpuLight.workQueue.size=mem / (2*3*32)
```

Comment:
Why not simply `cpuLight.queueSize`, are there, or will there be, other properties of the queue?